.github/workflows/_push-entrypoint.yaml

```
@@ -11,7 +11,8 @@ on:
       - 'e*'
     branches:
       - 'master'
-      - 'release-5?'
+      - 'release-53'
+      - 'release-54'
```

```suggestion
      - 'release-5[0-9]+'
```
(I think `?` matches the [preceding](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) char)

---

Thanks. I think `+` is not necessary.